### ISSUE TITLE
Single line ignore

### ISSUE THREAD

**Issue by matttrach** [hashicorp/vscode-terraform#281]
Is there a comment or something I could add to a line (in a Terraform file), which would disable warnings and errors for that line?

I am getting an error on a reference to a file that does not exist in the same repo as the one I am working on but will be deployed to the same server. I expect the error and because the linter does not have full context I can understand why the error exists, but if I could add a comment to disable the check for that single line it would prevent false negatives on my team.

**Comment by matttrach** [comment]
it would be better if I didn't have to commit anything to my repo to ignore the error, like if there is a plugin setting to tell it to ignore warnings and errors on the specified line.

**Comment by jmgilman** [comment]
In my case it's linting a reference to a Vault reference:

```
data "vault_generic_secret" "vsphere" {
  path = "/secret/vsphere/packer"
}

user = data.vault_generic_secret.vsphere.data["username"]
```

The linter gives the error:

> Unknown token: 10:20 IDENT data.vault_generic_secret.vsphere.data

It seems like a small incompatibility with the Vault integration, which is fine, but it would be nice to be able to ignore that specific line.

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
